src/webR/chan/channel.ts

```diff
@@ -69,6 +69,10 @@ export abstract class ChannelMain {
     return promise as Promise<WebRPayload>;
   }
 
+  protected outputClosedMessage(): void {
```

Comment:
A verb would be a little nicer?

```suggestion
  protected putClosedMessage(): void {
```